src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackRazorConfiguration.cs

```diff
@@ -58,6 +58,11 @@ public static RazorConfiguration SelectConfiguration(Version version)
             {
                 return MVC_3_0;
             }
+            else if (version.Major == 5 && version.Minor == 0)
```

Comment:
I couldn't find any tests for this. How should I verify this change?

Reply:
Do we need this, or can we rely on the existing implicit logic, which defaults to returning `Latest` which is already `MVC_3_0`? 

https://github.com/dotnet/aspnetcore-tooling/pull/2148/files/d8c3aca4929b596596ef1799ba0534f072ffba0b#diff-6c80c929d7c347ac32b93e86a79ae7feR37

Reply:
It might become problematic in the future if \ when we ship a 6.0 that uses a newer MVC extension.